theme/palette/accessible-variants.less

```diff
@@ -1,5 +1,7 @@
 // Accessible variants
 @sk-mid-gray-on-white: #959595;
+@sk-mid-gray-dark-on-gray-medium: #646464;
```

Comment:
As discussed this should be `@sk-mid-gray-dark-on-gray-light` as that is what the [background is set to](https://github.com/seek-oss/seek-style-guide/blob/master/theme/palette/elements.less#L10).